spec/CordovaCheck.spec.js

```diff
@@ -31,12 +31,12 @@ describe('findProjectRoot method', function () {
         process.chdir(cwd);
     });
     function removeDir (someDirectory) {
```

Comment:
I'd inline all occurrences of this helper and drop it, since it's now only an alias